### ISSUE TITLE
toggle_quick_menu does not save the config

### ISSUE THREAD

**Issue by A-Daneel** [ThePrimeagen/harpoon#49]
issue description: quick menu ordering does not update when using `toggle_quick_menu`. Using `:wq` does update it.

nvim version: 0.5.0-dev+1157-g0ab88c2ea
harpoon version: up to date as of 2021/03/27

`get_mark_config` before changing the order:
```
{                                                                                                            
  marks = { {
      col = 0,
      filename = "tests/file1.js",
      row = 43
    }, {
      col = 0,
      filename = "tests/file2.js",
      row = 19
    }, {
      col = 0,
      filename = "tests/data.json",
      row = 81
    }, {
      col = 0,
      filename = "tests/data.json_backup",
      row = 23
    } }
}
```

after changing the 2nd and 3th item around and close using `toggle_quick_menu` nothing has changed.

**Comment by asbjornhaland** [comment]
@ThePrimeagen: Can be closed after #50, right?

**Comment by ThePrimeagen** [comment]
I am going to close it because #50 should of addressed it.
